src/ui/component/publishForm/view.jsx

```diff
@@ -87,6 +92,45 @@ function PublishForm(props: Props) {
     submitLabel = !publishing ? __('Publish') : __('Publishing...');
   }
 
+  const success = successResponse => {
```

Comment:
Instead of bringing this into the component, we can still use a `doPublish` function that lives inside of the app. All it will do is call the publish function from lbry-redux, and pass in this success callback.